src/v2/Apps/Consign/Components/InDemandNow/ConsignTopArtists.tsx

```diff
@@ -85,11 +90,53 @@ const TopArtists: React.FC<ConsignTopArtistsQuery["response"]> = props => {
     return null
   }
 
+  // Iterate over microfunnel artworks and a) group into "chunks" of four for
+  // displaying in rows; b) average each artwork realized price; c) shuffle the
+  // display so that its a bit less static.
   const recentlySoldArtworks = chunk(
     shuffle(
-      microfunnelItems.map(
-        artwork => artwork?.artworksConnection?.edges?.[0]?.node
-      )
+      microfunnelItems.map(artwork => {
+        if (artwork?.artworksConnection?.edges?.length === 0) {
+          return null
+        }
+
+        const { edges } = artwork.artworksConnection
+
+        const realizedPriceAverage = edges.reduce(
+          (artworksWithPrice, artwork, index, artworks) => {
+            const realizedPrice = Number(
+              // Value comes back from MP formatted; strip dollars and commas
+              artwork.node.realizedPrice.replace("$", "").replace(",", "")
+            )
+
+            // Some artworks don't have a realized price, and return 0. We only
+            // want to average up non-zero artworks
+            if (realizedPrice) {
+              artworksWithPrice.push(realizedPrice)
+            }
+
+            if (index === artworks.length - 1) {
+              const totalValue = artworksWithPrice.reduce(
+                (total, price) => total + price,
+                0
+              )
+              const average = totalValue / artworksWithPrice.length
+              const averageValueInDollars = formatCentsToDollars(100 * average)
+              return averageValueInDollars
+            }
+            return artworksWithPrice
+          },
+          []
+        )
```

Comment:
@williardx - had to update this a bit... Was noticing that the numbers seem pretty low for top tier artists and realized that we were dividing the total by the number of works -- even if the artwork didn't have a realized price. So now just pushing items that we do have prices into an array and dividing by that total. 

Look good?